src/cosmic/cosm.rs

```diff
@@ -1396,7 +1396,7 @@ mod tests {
             Bodies::EarthBarycenter.ephem_path(),
             jde,
             mars2k,
-            LightTimeCalc::Abberation,
+            LightTimeCalc::Aberration,
```

Comment:
Same comment

Reply:
Resolved in commit 96a6ad32a5d0dc19579112d430d2411efecbbf96.